aten/src/ATen/cuda/CUDAEvent.h

```diff
@@ -157,6 +157,10 @@ struct TORCH_CUDA_CPP_API CUDAEvent {
   // Note: cudaEventSynchronize can be safely called from any device
   void synchronize() const {
     if (is_created_) {
+      const c10::impl::PyInterpreter* interp = c10::impl::GPUTrace::get_trace();
```

Comment:
I wonder, why isn't there any code related to the "sync debug mode" before this call?

Reply:
I've been wondering about that as well. This seems like a place where it could be added.